app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = False
+        self.alive.append(self)
```

Comment:
please read the checklist

Reply:
Animal.alive instead of self.alive, because self - reference for the instance of the class, Animal - class by itself